src/components/ViewSwitch.tsx

```diff
@@ -19,23 +20,35 @@ enum View {
 
 const ViewSwitch: React.FC = () => {
     const [currentView, setCurrentView] = useState<View>(View.Lab);
+    const [previousModule, setPreviousModule] = useState<Module>(Module.A_B_AB);
 
     const switchView = () => {
         setCurrentView((prevView) =>
             prevView === View.Lab ? View.Simulation : View.Lab
         );
     };
-    const { page, isPlaying, setIsPlaying, handleTimeChange } =
+    const { page, isPlaying, setIsPlaying, handleTimeChange, module } =
         useContext(SimulariumContext);
 
-    usePageNumber(
-        page,
-        (page) => page === 1 && currentView === View.Simulation,
-        () => {
-            // reset home
-            setCurrentView(View.Lab);
+    const isFirstPageOfFirstModule =
+        page === FIRST_PAGE && module === Module.A_B_AB;
+
+    if (isFirstPageOfFirstModule && currentView === View.Simulation) {
+        setCurrentView(View.Lab);
+    }
+
+    const { contentData } = useModule(module);
+
+    // The first time the user gets to the second module
+    // it should show the simulation view
```

Comment:
Does this really only show the first time? Or just anytime the user switches modules to the second module?